src/Renci.SshNet/SshCommand.cs

```diff
@@ -315,7 +320,9 @@ public string EndExecute(IAsyncResult asyncResult)
 
                 commandAsyncResult.EndCalled = true;
 
+#pragma warning disable CS0618
```

Comment:
Not sure how to best handle this, but since this is obsolete, we have to stop the compiler from giving a warning here as the project will not compile otherwise.

Reply:
We could have private GetResult() that would be called from here and from Result.